src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -222,5 +238,34 @@ export default class HexagonLayer extends Layer {
   }
 }
 
+// if hexagon vertices not provided, calculate a 1km radius hexagon
+// at the first object's centroid location
+function _getDefaultVertices(data, getCentroid) {
```

Comment:
Again I would have used the provided radius (in meters) and angle here to generate verts, rather than just assuming 1 kilometer.

The Viewport should have distance mapping functions now that convert meters to lnglat etc. This would be a good test to make sure they work the same in JS and GLSL.